runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -187,4 +238,98 @@ private void restoreOriginalDots()
 
 		System.arraycopy(originalDotSprites, 0, mapDots, 0, mapDots.length);
 	}
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.MINIMAP_GROUP_ID)
+		{
+			if (config.leftAlignOrbs() != AlignmentMode.OFF)
+			{
+				if (config.leftAlignOrbs() == AlignmentMode.BOTH)
+				{
+					moveOrbs();
+				}
+				else if (!client.isResized() && config.leftAlignOrbs() == AlignmentMode.FIXED)
+				{
+					moveOrbs();
+				}
+				else if (client.isResized() && config.leftAlignOrbs() == AlignmentMode.RESIZEABLE)
+				{
+					moveOrbs();
+				}
+			}
+		}
+	}
+
+	private void moveOrbs()
+	{
+		Widget healthOrb = client.getWidget(WidgetInfo.MINIMAP_HEALTH_ORB);
+		Widget prayerOrb = client.getWidget(WidgetInfo.MINIMAP_PRAYER_ORB);
+		Widget runOrb = client.getWidget(WidgetInfo.MINIMAP_RUN_ORB);
+		Widget specOrb = client.getWidget(WidgetInfo.MINIMAP_SPEC_ORB);
+
+		if (healthOrb != null)
+		{
+			healthOrb.setRelativeY(33);
```

Comment:
Can these numbers be calculated?

Reply:
They're 32 above the previous one, except for the first one which is 33, but it could probably be 32 and still be fine.